pyqt5_ui.py

```diff
@@ -0,0 +1,102 @@
+import sys
+from PyQt5.QtWidgets import QApplication, QMainWindow, QFrame, QMessageBox
+from game import Game
+from PyQt5.QtGui import QPainter, QColor
+
+
+class CrushUI(QMainWindow):
+    def __init__(self):
+        super().__init__()
+        red = (255, 0, 0)
+        yellow = (250, 223, 0)
+        light_green = (0, 255, 127)
+        navy = (0,0,128)
+        blue = 	(0,0,255)
+        lavender = (221, 160, 221)
+        aqua = (0, 255, 255)
+        green = (0, 128, 0)
+        maroon = (139, 0, 0)
+        game.characters = [aqua, green, light_green, yellow, red, navy, lavender, maroon, blue]
+        game.set_up_game()
+        self.set_up_widget()
+
+    def set_up_widget(self):
+        width, height = 500, 500
+        self.resize(width, height)
+        board_widget = CrushWidget(self)
+        self.setCentralWidget(board_widget)
+        self.setWindowTitle('Crush It!')
+        self.status_bar = self.statusBar()
+        self.refresh_status_bar()
+        self.show()
+        QMessageBox.information(self, 'Information', 'Crush at least 3 Items!')
+
+    def refresh_status_bar(self):
+        self.status_bar.showMessage('Score: {}                   Target: {}'.format(game.score * 10,
+                                                                                    game.target * 10))
+
+
+class CrushWidget(QFrame):
+    def __init__(self, parent):
+        super().__init__()
+        self.parent = parent
+        self.set_size()
+
+    def set_size(self):
+        self.height = self.parent.frameGeometry().height() - 49
```

Comment:
what is this 49 number?

Reply:
This was because I did some unnecessary math to calculate the frame-size. The 49 was the height of the status bar. I changed it so now I just use QFrame.height()